### PR TITLE
Changes for MS PAPI Expansion

### PR DESCRIPTION
**Pull request by JasperLorelai**
Changes:
- Added skip in Spell#initSubspell().
- Added some changes to support more placeholders in the PAPI MagicSpells Expansion.
- Fix for PlayerMenuSpell.

### CODE REVIEW

File: core/src/main/java/com/nisovin/magicspells/variables/GlobalVariable.java [hunk 1]
```diff
@@ -37,5 +37,14 @@ public double getValue(String player) {
 	public void reset(String player) {
 		value = defaultValue;
 	}
-	
+
+	@Override
+	public double getMaxValue() {
```
Comment (TheComputerGeek2) [inline core/src/main/java/com/nisovin/magicspells/variables/GlobalVariable.java]: What is the reason for this? Doesn't the definition of the max value exist within a superclass?

File: core/src/main/java/com/nisovin/magicspells/variables/GlobalVariable.java [hunk 2]
```diff
@@ -37,5 +37,14 @@ public double getValue(String player) {
...
+		return maxValue;
+	}
+
+	@Override
+	public double getMinValue() {
```
Comment (TheComputerGeek2) [inline core/src/main/java/com/nisovin/magicspells/variables/GlobalVariable.java]: Same as max value comment

File: core/src/main/java/com/nisovin/magicspells/variables/PlayerVariable.java [hunk 3]
```diff
@@ -44,5 +44,14 @@ public void reset(String player) {
 		map.remove(player);
 		if (objective != null) objective.getScore(Bukkit.getOfflinePlayer(player)).setScore((int) defaultValue);
 	}
-	
+
+	@Override
+	public double getMaxValue() {
```
Comment (TheComputerGeek2) [inline core/src/main/java/com/nisovin/magicspells/variables/PlayerVariable.java]: See comment on GlobalVariable

File: core/src/main/java/com/nisovin/magicspells/variables/PlayerVariable.java [hunk 4]
```diff
@@ -44,5 +44,14 @@ public void reset(String player) {
...
+		return maxValue;
+	}
+
+	@Override
+	public double getMinValue() {
```
Comment (TheComputerGeek2) [inline core/src/main/java/com/nisovin/magicspells/variables/PlayerVariable.java]: See comment on GlobalVariable

File: core/src/main/java/com/nisovin/magicspells/variables/GlobalVariable.java (hunk 1 above)
Comment (JasperLorelai) [reply core/src/main/java/com/nisovin/magicspells/variables/GlobalVariable.java]: It does. I didn't see a way to access either, so I made these methods. If I had made them `public`, would that have fixed the issue? Or is there a different solution to this?

**Comment by JasperLorelai** [comment]
Added commit that should resolve this.

**Pull request merged**